samples/src/Microsoft.Azure.IIoT.App/src/Services/Browser.cs

```diff
@@ -330,17 +330,21 @@ public class Browser {
             model.Header = Elevate(new RequestHeaderApiModel(), credential);
 
             try {
-                var count = 0;
-                foreach (var item in parameters.InputArguments) {
-                    var argument = new MethodCallArgumentApiModel {
-                        Value = parameterValues[count] ?? string.Empty,
-                        DataType = item.Type.DataType
-                    };
-                    argumentsList.Add(argument);
-                    count++;
+                if (parameters.InputArguments != null)
+                {
```

Comment:
Please do not disable .editorconfig so we have single formatting.  (see line breaks)

Reply:
my formatting config was screwed up. Fixed